### PR TITLE
Send Update Queries to seperate endpoint

### PR DESCRIPTION
**Pull request by nck-mlcnv**

### CODE REVIEW

File: src/test/java/org/aksw/iguana/cc/query/QueryDataTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    @BeforeAll
+    public static void setup() throws IOException {
+        tempFile = Files.createTempFile("test", "txt");
+        Files.writeString(tempFile, """
+                SELECT ?s ?p ?o WHERE {
+                    ?s ?p ?o
+                }
+                
+                INSERT DATA {
```
Comment (bigerl) [inline src/test/java/org/aksw/iguana/cc/query/QueryDataTest.java:30]: Just realized we are also missing something here. Please create an issue to also support multiple semicolon-chained update operations. 

https://www.w3.org/TR/2013/REC-sparql11-update-20130321/#updateLanguage

**Review by bigerl — APPROVED** [review]
Thx for the PR. This looks already quite polished. Please just create the issue metioned below.

File: src/test/java/org/aksw/iguana/cc/query/QueryDataTest.java (hunk 1 above)
Comment (nck-mlcnv) [reply src/test/java/org/aksw/iguana/cc/query/QueryDataTest.java:30]: What exactly is the problem with semicolon-chained update operations? Should they be split up into multiple queries?

Comment (bigerl) [reply src/test/java/org/aksw/iguana/cc/query/QueryDataTest.java:30]: Good question. They should just work, right? Only, We should probably double check that LSQ handles them correctly when we include that.

**Review by bigerl — APPROVED** [review]

**Pull request merged**